.github/dependabot.yml

```diff
@@ -0,0 +1,15 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "gradle" # See documentation for possible values
+    directory: "/" # Location of package manifests
```

Comment:
Should this be "/library"

Reply:
I would avoid this. Look at https://dependabot.com/java/

<img width="458" alt="image" src="https://user-images.githubusercontent.com/3314607/112197307-3b75bc80-8c0c-11eb-826d-01cdacf4bc53.png">

This pull request has more then current master, which doesn't work at all.

If you observe a missing scan, you can add it anytime.

Reply:
Here is the evidence https://github.com/hannesa2/mlkit/pull/24/files